packages/builders/src/interactions/slashCommands/mixins/ApplicationCommandOptionChannelTypesMixin.ts

```diff
@@ -1,5 +1,5 @@
 import { ChannelType } from 'discord-api-types/v9';
-import { z, ZodLiteral } from 'zod';
+import { s, LiteralValidator } from '@sapphire/shapeshift';
```

Comment:
```suggestion
import { s } from '@sapphire/shapeshift';
```
The cast this is used in is no longer needed.